src/FtxExchangeConfiguration.ts

```diff
@@ -44,6 +47,45 @@ export class FtxExchangeConfiguration implements ExchangeConfiguration {
     }
   }
 
+  fetchDepositsValidateInput(address: string, amountInSats: number) {
+    assert(address, ApiError.UNSUPPORTED_ADDRESS)
+    assert(amountInSats, ApiError.NON_POSITIVE_QUANTITY)
+    assert(amountInSats > 0, ApiError.NON_POSITIVE_QUANTITY)
+  }
+  fetchDepositsProcessApiResponse(
+    address: string,
+    amountInSats: number,
+    response,
+  ): FetchDepositsResult {
+    assert(response, ApiError.UNSUPPORTED_API_RESPONSE)
+    const amountInBtc = sat2btc(amountInSats)
+    // We should receive an array of array of response objects
+    // Since we're looking for one specific entry, don't fail until we're done
+    const result = {} as FetchDepositsResult
+    response.forEach((responses) => {
```

Comment:
response and responses are probably inverted here

Reply:
poor naming, updated with deposits